sbt-mode.el

```diff
@@ -176,6 +176,11 @@ buffer called *sbt*projectdir."
         (message "Starting sbt in buffer %s " buffer-name)
         ;;(erase-buffer)
 
+	;; In windows set SBT_OPTS like "-Djline.terminal=jline.UnsupportedTerminal"
+	;; https://github.com/ensime/emacs-sbt-mode/issues/44
+	(when (and (eql system-type 'windows-nt) (not (getenv "SBT_OPTS")))
+	  (setenv "SBT_OPTS" "-Djline.terminal=jline.UnsupportedTerminal"))
```

Comment:
this will override anything the user has seen, you need to add to an existing envvar.